packages/verification-vue-component/src/components/CertifactionVerification.vue

```diff
@@ -276,19 +291,7 @@ export default {
                         identityVerifier.image = offchainVerification.issuerVerifiedImg
                     }
 
-                    if (!verification.events && offchainVerification.status === 'registering') {
-                        // File not found on blockchain and offchain status is registering
-                        verification = {
-                            ...verification,
-                            ...offchainVerification,
-                            // TODO(Cyrill): Remove when verify endpoint returns an events array
-                            events: [{
-                                scope: 'register',
-                                issuer: offchainVerification.issuerName,
-                                identityVerifier: (Object.keys(identityVerifier).length > 0) ? identityVerifier : null
-                            }]
-                        }
-                    } else if (['registered', 'registering', 'revoking', 'revoked'].indexOf(offchainVerification.status) >= 0) {
+                    if (verification.events && verification.events.length > 0 && offchainVerification.status !== 'registering') {
```

Comment:
Can we use some sorts of enums for the statuses?

Reply:
Would be possible, but there is only one other place where this status is checked.